docs/wall.html

```diff
@@ -0,0 +1,225 @@
+                <!-- <li><a href="foo.html">overview</a></li> -->
+                <li><a href="./index.html">home</a></li>
+                <li><a class="grey-text" >leaderboard</a></li>
+                <li>
+                  <a href="https://github.com/MemeInvestor/memeinvestor_bot/" target="_blank">
+                  <img class="icon-32" src="resources/GitHub-Mark-Light-32px.png"/></a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+
+    <ul class="sidenav" id="mobile-sidebar">
+        <li class="sidenav-top">
+         <h5>menu</h5>
+         <a class="sidenav-close" href="#!"><i class="material-icons">close</i></a>
+        </li>
+        <li><a id="without-this-everything-brokes">lol</a></li>
+        <li class="sidebar-search">
+            <input id="investor-username-mobile" placeholder="search user" type="search"  >
+        </li>
+        <!-- <li><a href="foo.html">brokers</a></li> -->
+        <li><a href="index.html">home</a></li>
+        <li><a class="grey-text" >leaderboard</a></li>
+        <li><a href="https://github.com/MemeInvestor/memeinvestor_bot/">
+        <img class="icon-32" src="resources/GitHub-Mark-Light-32px.png" target="_blank"></a></li>
+    </ul>
+    
+    <div id="announcements-modal" class="modal ">
+    </div>
+
+<!-- PUT PAGE CODE HERE -->
+
+    <div class="container" >
+      <div class="section center-align">
+        <h4 class="no-margin-top" >Top Traders wall</h4>
+        <p>Select one of the market seasons below to view it's top 50 traders.</p>
+        <a id="beta-bt" class="waves-effect waves-light btn grey darken-3 disabled">beta</a>
+    <!--<a href="#info1" class="waves-effect waves-light btn grey darken-3 disabled"><i class="material-icons right">expand_more</i>select</a> -->
+        <a id="current-bt" class="waves-effect waves-light btn grey darken-3">current</a>
+        
+      <div class="row top3-cards">
+        <br>
+        <div class="top-card">
+          <img src="resources/first.png">
+          <h5 id="top-name">-</h5>
+          <p><span id="top-net-worth">0</span> <span id="top-net-worth-suffix"></span><br>
+          <a id="top-profile">profile</a></p>     
+        </div>   
+    
+        <div class="col s12 m6 l6 left-card">
+          <div class="divider hide-on-med-and-up"></div>
+          <img src="resources/2.png">
+          <h5 id="left-name">-</h5>
+          <p><span id="left-net-worth">0</span> <span id="left-net-worth-suffix"></span><br>
+          <a id="left-profile">profile</a></p> 
+        </div>
+        
+        <div class="col s12 m6 l6 right-card">
+          <div class="divider hide-on-med-and-up"></div>
+          <img src="resources/3.png">
+          <h5 id="right-name">-</h5>
+          <p><span id="right-net-worth">0</span> <span id="right-net-worth-suffix"></span><br>
+          <a id="right-profile">profile</a></p> 
+        </div>
+        
+
+        
+      </div>
+      
+      <div class="table-wrap" >
```

Comment:
Can we clean up the sizing of the leaderboards columns? On my laptop (using Edge browser) there's a bunch of empty space to the right of "Completed investments". Looks like the table should be about 75% of its current width.

Reply:
@robalb Sent to you on Slack.